### PR TITLE
moving envify to dependencies

### LINKED ISSUE #31: Error when using w/ RequireBin
RequireBin is a JSFiddle-alike that automatically browserifies require() calls using BrowserifyCDN.

When I try to use this package with it I get an error about 'envify'. 

screen shot 2015-10-07 at 4 20 24 pm

A little research shows that [React had similar problems a while back](https://github.com/facebook/react/issues/1482) because they listed envify as a `peerDependency`.

### PR DESCRIPTION
**Pull request by jessehattabaugh**
fixes #31

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "planet-client",
-  "version": "0.14.0",
+  "version": "0.15.1",
```
Comment (tschaub) [inline package.json]: Please don't update the published version in a pull request.  This lets us merge multiple branches per release.

**Comment by tschaub** [comment]
Thanks for digging into this @jessehattabaugh.  You've got conflicts to resolve related to updating the version number in a feature branch.  There are two sensible options for updating version numbers in `package.json`:
- create a dedicated release branch, tag and publish from the release branch
- update version numbers in master only, tag and publish from master

I think the latter is a lighter weight process.  For larger projects with more involved releases, the former can be appropriate.  It is never really appropriate to bump the version number in a feature branch.

If you can rework this so it can be merged, that would be great.  All this could be squashed into a single commit that moves `envify` from `devDependencies` to `dependencies` (no need for the experimental commits in the history).

File: package.json (hunk 1 above)
Comment (jessehattabaugh) [inline package.json]: makes sense, I'll keep it in mind in the future

**Comment by tschaub** [comment]
@jessehattabaugh hope this didn't step on your toes too much.  I rebased and squashed all your work into one commit.

Previous history:
- 9e88df99805963e21316f613c9492e9358f4acec
- 37a48225fc28551607b4181b566da04d209dbb28
- 9d0e6b1292e952c93709cc86e3b276dbceb92493
- 44c020d3c4f632609c132ebea9357f0ce5ba7166
- 7f9c1ff44f918a160fe24a901f0edaa9d9916d38
- 53ae6d9090881ad97b855cd7505f075e032b49d8

New history:
- db7c5527596b45f84338cf18301d9b8dc66cdd53

**Pull request merged**